src/LineageTree/lineageTree.py

```diff
@@ -2140,9 +2130,9 @@ def norm(*args):
         delta_tmp = partial(
             delta, corres1=corres1, corres2=corres2, times=self.cycle_time
         )
-        return uted(nodes1, adj1, nodes2, adj2, delta=delta_tmp) / norm(
-            nodes1, nodes2
-        )
+        return uted(
+            nodes1, adj1, nodes2, adj2, delta=delta_tmp
+        )   / max(len(self.get_sub_tree(n1)),len(self.get_sub_tree(n2)))
```

Comment:
Here the norm function that I had in place allowed you to compute the max between the two subtrees (see L2117).
Have a function which defaults to the max but can be changed allows for more flexibility.